CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [1.0.0] - 2019-02-20
+First version of the extension.
+### Added
+- Added support to display pdf files passed through properties on pdp pages.
+- Configurations include sectionTitle, manualsProperty, manualsPrefixUrl, typeIndices, and propertySplit.
+
+[Link to Diff](https://github.com/shopgate-professional-services/ext-pdp-manual-section/compare/v0.0.1...v1.0.0)
```

Comment:
Please adjust the changelog to our format.

Reply:
Updated to reflect format. Will add tag when approved